tools/ipapy2/ipapy2_gibbs_sampler.xml

```diff
@@ -0,0 +1,109 @@
+        <conditional name="integrating_mode">
+            <param name="integrating_mode" type="select" label="integrating mode" help="select the integrating mode">
+                <option value="adducts">adducts</option>
+                <option value="biochemical">biochemical</option>
+                <option value="both">adducts and biochemical</option>
+            </param>
+            <when value="adducts">
+                <param name="delta_add" type="float" value="1" min="0" label="adducts weight" 
+                        help="parameter used when computing the conditional priors. The parameter must be positive. 
+                        The smaller the parameter the more weight the adducts connections have on the posterior probabilities. Default 1." />
+            </when>
+            <when value="biochemical">
+                <param name="Bio" type="data" format="csv" label="biochemical connections" 
+                        help="dataframe (2 columns), reporting all the possible connections between compounds. It uses the unique ids from the database. 
+                        It could be the output of Compute_Bio() or Compute_Bio_Parallel()." />
+                <param name="delta_bio" type="float" value="1" min="0" label="biochemical weight" 
+                        help="parameter used when computing the conditional priors. The parameter must be positive. 
+                        The smaller the parameter the more weight the biochemical connections have on the posterior probabilities. Default 1." />            
+            </when>
+            <when value="both">
+                <param name="delta_add" type="float" value="1" min="0" label="adducts weight" 
+                        help="parameter used when computing the conditional priors. The parameter must be positive. 
+                        The smaller the parameter the more weight the adducts connections have on the posterior probabilities. Default 1." />
+                <param name="Bio" type="data" format="csv" label="biochemical connections" 
+                        help="dataframe (2 columns), reporting all the possible connections between compounds. It uses the unique ids from the database. 
+                        It could be the output of Compute_Bio() or Compute_Bio_Parallel()." />
+                <param name="delta_bio" type="float" value="1" min="0" label="biochemical weight" 
+                        help="parameter used when computing the conditional priors. The parameter must be positive. 
+                        The smaller the parameter the more weight the biochemical connections have on the posterior probabilities. Default 1." /> 
+            </when>
+        </conditional>
+    </inputs>
+
+    <outputs>
+        <data label="${tool.name} annotations on ${on_string}" name="annotations_out" format="csv"/>
+        <data label="${tool.name} zs on ${on_string}" name="zs_out" format="csv">
+            <filter>options['gibbs_out']</filter>
```

Comment:
Not sure if this works with options as there is no section defined for this parameter.